megalinter/descriptors/arm.megalinter-descriptor.yml

```diff
@@ -63,23 +63,21 @@ linters:
   # Bicep Linter
   - name: ARM_BICEP_LINTER
     config_file_name: bicepconfig.json
-    cli_help_extra_args:
-      - "--help"
-    cli_lint_mode: "file"
-    cli_lint_extra_args:
-      - "-f"
-    cli_version_arg_name: "version"
+    cli_help_extra_args: --help
```

Comment:
are you sure it's not cli_help_arg_name that you want to use here ? :)

Reply:
that would make more sense...